segmentation/utils/data_utils_test.py

```diff
@@ -27,27 +28,30 @@ def _create_img_dir(temp_dir, fovs, imgs, img_sub_folder="TIFs"):
 
 
 def test_validate_paths():
-    valid_path = '.'
 
-    valid_parts = [p for p in pathlib.Path(valid_path).resolve().parts]
-    valid_parts[1] = 'not_a_real_directory'
+    valid_path = 'data/example_dataset/label_data'
```

Comment:
This test will fail if we decide to reorganize the directory. Let's create a synthetic directory to use during testing with tmpdir.